packages/core/libs/createBlockClass.ts

```diff
@@ -44,6 +44,7 @@ export function createBlockClass (type: TBlockType, id: string, props: Omit<Nish
 		case 'quote':
 		case 'divider':
 		case 'callout':
+		case 'embed':
```

Comment:
I think this should be allowed. There is a type for it already: "IEmbedInput"

Reply:
Yes thank you for that